cpp/tests/test_parameter_studies.cpp

```diff
@@ -44,11 +44,11 @@ TEST(ParameterStudies, sample_from_secir_params)
 
     auto& params = model.parameters;
     for (auto i = mio::Index<mio::AgeGroup>(0); i.get() < (size_t)num_groups; i++) {
-        params.get<mio::osecir::IncubationTime>()[i]       = 5.2;
-        params.get<mio::osecir::TimeInfectedSymptoms>()[i] = 5.;
-        params.get<mio::osecir::SerialInterval>()[i]       = 3.9;
-        params.get<mio::osecir::TimeInfectedSevere>()[i]   = 10.;
-        params.get<mio::osecir::TimeInfectedCritical>()[i] = 8.;
+        params.get<mio::osecir::TimeExposed>()[i]            = 3.2;
```

Comment:
Serial interval was 3.9 before so we would have TimeExposed=2.6 and TimeInfectedNoSyptoms=2.6. Does that make a difference?